COC_PyBay_Response.md

```diff
@@ -0,0 +1,66 @@
+#Code of Conduct Response Playbook
+
+This document aims to help organizers and moderators of PyBay spaces to handle reports of Code of Conduct violations.
+
+###COC VIOLATIONS AT CONFERENCES
+
+###Presentations
+
+Presentations or similar events should not be stopped for one-time gaffes or minor problems, although a member of conference staff should speak to the presenter afterward. However, staff should take immediate action to politely and calmly stop any presentation or event that repeatedly or seriously violates the anti-harassment policy. For example, simply say "I’m sorry, this presentation cannot be continued at the present time" with no further explanation.
```

Comment:
What does the word "simply" add to the explanation here?

Reply:
removed